typescript-conditionals/main.ts

```diff
@@ -0,0 +1,104 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+// Start exercise//
+
+// isUnderFive(number)
+
+const isUnderFive = (number: number): boolean => {
+  return number < 5;
+};
+
+// isEven(number)
+
+const isEven = (number: number): boolean => {
+  return (number & 2) === 0;
+};
+
+// startWithJ(string)
+
+const startsWithJ = (string: string): boolean => {
+  return string.charAt(0) === 'J';
+};
+
+// isOldEnoughToDrink(person)
+
+interface Person {
+  name: string;
+  age: number;
+}
+
+const isOldEnoughToDrink = (person: Person): boolean => {
+  return person.age >= 21;
+};
+
+const isOldEnoughToDrive = (person: Person): boolean => {
+  return person.age >= 16;
+};
+
+const isOldEnoughToDrinkAndDrive = (): boolean => {
+  return false;
```

Comment:
They want to use AND operator to combine the condition for drive and condition for drink here